src/policy/fees.h

```diff
@@ -225,19 +225,19 @@ class CBlockPolicyEstimator
     void processBlock(unsigned int nBlockHeight,
                       std::vector<CTxMemPoolEntry>& entries, bool fCurrentEstimate);
 
-    /** Process a transaction confirmed in a block*/
+    /** Process a transaction confirmed in a block */
```

Comment:
Again, doxygen doesn't care about such white space and I don't think this diff is required.